src/components/HeroHeader/index.jsx

```diff
@@ -54,11 +49,7 @@ const HeroHeader = ({ t, client, breakpoints: { isMobile } }) => {
 }
 
 HeroHeader.propTypes = {
-  t: PropTypes.func.isRequired,
-  client: PropTypes.object.isRequired,
-  breakpoints: PropTypes.shape({
-    isMobile: PropTypes.bool
-  }).isRequired
+  client: PropTypes.object.isRequired
```

Comment:
AMHA, there is no need to keep the `client` propType as it is injected via the HOC.